changelog/519.misc.rst

```diff
@@ -0,0 +1 @@
+Run pyright on examples.
```

Comment:
This isn't really descriptive. Running pyright on its own doesn't change any files.